src/.eslintrc

```diff
@@ -0,0 +1,11 @@
+---
+extends: ../node_modules/builder-docs-archetype/config/eslint/.eslintrc-react
```

Comment:
No, doesn't seem to resolve. Pointing to nodemods is the only path that works

`Error: Cannot read config package: eslint-config-builder-docs-archetype/config/eslint/.eslintrc-config`